python-package/xgboost/__init__.py

```diff
@@ -29,4 +30,5 @@
            'RabitTracker',
            'XGBModel', 'XGBClassifier', 'XGBRegressor', 'XGBRanker',
            'XGBRFClassifier', 'XGBRFRegressor',
-           'plot_importance', 'plot_tree', 'to_graphviz', 'dask']
+           'plot_importance', 'plot_tree', 'to_graphviz', 'dask',
+           'set_config', 'get_config', 'config_context']
```

Comment:
Is it necessary to have explicit getter and setter?

Reply:
I was following sklearn's config manager: https://scikit-learn.org/stable/modules/generated/sklearn.config_context.html

Reply:
Also, the context manager is implemented in terms of the getter and the setter.